docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
 
-    **Note:** Observe the following behaviors:
+###Applications Distributed per Region
 
-    - The application deployment process may take up to a few minutes, depending on the size of container images to be pulled, the target locations, and the number of expected instances. Once the application is deployed, **Deployed** appears as its status on the Applications page. To check the status and other details, use the applications list view and detail view. After the application is deployed, you can retrieve instances in [list view](</docs/portal/applications/retrieving-instances-of-an-application.md#retrieving-instances-of-an-application-in-list-view>) or [map view](</docs/portal/applications/retrieving-instances-of-an-application.md#retrieving-instances-of-an-application-in-map-view>).
-    - Updating an application using the method PUT replaces all existing objects, including any persistent storage volumes, of the application.
-    - Scaling down or redistributing an application that uses persistent storage volumes might delete existing persistent volumes along with pod instances.
\ No newline at end of file
+If your application is distributed per region, a dialog box similar to the following appears.
+
+<<new_Redistribute-Your-Application-Distribute-per-Region.png goes here>>
+
+1. Under **Targets**, accept or change the regions and replicas per region shown for existing targets.
+2. To add targets, click the **+ Add Target** button. When a new row appears at the bottom of **Targets**, select a region and enter the number of replicas for that region.
+3. To remove a target, click the trash can icon at the right side of the target.
+4. When you finish, click **Redistribute**.
+
+###Applications Distributed per Pop
+
+If your application is distributed per PoP, a dialog box similar to the following appears.
+
+<<new_Redistribute-Your-Application-Distribute-per-Pop.png goes here>>
+
+1. Under **Targets**, accept or change the PoPs and replicas per PoP shown for existing PoPs.
```

Comment:
I believe you intended "existing targets" rather than "existing PoPs".

Reply:
Fixed.